Packs/ShiftManagement/Playbooks/Shift_handover.yml

```diff
@@ -0,0 +1,1663 @@
+      id: 61e1375b-0419-4b19-804a-ee861d967685
+      version: -1
+      name: Get OOO users and clean OOO list
+      description: Retrieves users that are currently out of office.
+      scriptName: GetUsersOOO
+      type: regular
+      iscommand: false
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "6"
+    scriptarguments:
+      listname: {}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 480,
+          "y": -670
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "28":
+    id: "28"
+    taskid: 826aa6e4-1259-44ea-85f7-187b161301ca
+    type: regular
+    task:
+      id: 826aa6e4-1259-44ea-85f7-187b161301ca
+      version: -1
+      name: Search for Incidents
+      description: Searches Cortex XSOAR incidents.
+      scriptName: SearchIncidentsV2
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "29"
+    scriptarguments:
+      details: {}
+      fromclosedate: {}
+      fromdate: {}
+      fromduedate: {}
+      id: {}
+      level: {}
+      name: {}
+      notstatus: {}
+      owner: {}
+      page: {}
+      query:
+        simple: ${inputs.IncidentSearchQuery}
+      reason: {}
+      size: {}
+      sort: {}
+      status: {}
+      toclosedate: {}
+      todate: {}
+      toduedate: {}
+      type: {}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 660,
+          "y": 930
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "29":
+    id: "29"
+    taskid: 0beb2ecd-fc46-44f9-800f-125b89ffdd7f
+    type: regular
+    task:
+      id: 0beb2ecd-fc46-44f9-800f-125b89ffdd7f
+      version: -1
+      name: Add open incidents to layout
+      description: Creates a grid table from items or key-value pairs.
+      scriptName: SetGridField
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "19"
+    scriptarguments:
+      columns:
+        simple: Incident ID,Incident name,Incident owner,Link to incident
+      context_path:
+        simple: foundIncidents
+      grid_id:
+        simple: shiftopenincidents
+      keys:
+        simple: id,name,owner,incidentLink
+      overwrite: {}
+      sort_by: {}
+      unpack_nested_elements: {}
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 660,
+          "y": 1090
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  "30":
+    id: "30"
+    taskid: 26c61842-7348-4d51-8c9f-bfa1915a552f
+    type: regular
+    task:
+      id: 26c61842-7348-4d51-8c9f-bfa1915a552f
+      version: -1
+      name: 'Set apps Channel, team name and message to incident field '
+      description: commands.local.cmd.set.incident
```

Comment:
@ShirleyDenkberg this is a server default description and not the correct description.